example/20-componentconfig-gardener-scheduler.yaml

```diff
@@ -10,11 +10,16 @@ leaderElection:
   renewDeadline: 10s
   retryPeriod: 2s
   resourceLock: leases
+  resourceNamespace: garden
 logLevel: info
-server:
+healthServer:
   http:
     bindAddress: 0.0.0.0
     port: 10251
+metricsServer:
+  http:
+    bindAddress: 0.0.0.0
+    port: 19251
```

Comment:
Please also adapt https://github.com/gardener/gardener/tree/master/charts/gardener/controlplane/charts/runtime/templates/scheduler based on the API changes.